packages/location-management/src/components/LocationUnitGroupView/tests/__snapshots__/table.test.tsx.snap

```diff
@@ -0,0 +1,71 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`location-management/src/components/LocationTagView Test Table Delete 1`] = `null`;
+
+exports[`location-management/src/components/LocationTagView renders without crashing 1`] = `
+Object {
+  "children": <Router
```

Comment:
We can shorten this snapshot by targetting only the component of interest, we do not need to include `Router` in the snapshot

Reply:
done